client/src/components/annotators/ImageAnnotator.vue

```diff
@@ -312,10 +312,16 @@ export default defineComponent({
       const imgInternal = cacheFrame(0);
       imgInternal.onloadPromise.then(() => {
         initializeViewer(imgInternal.image.naturalWidth, imgInternal.image.naturalHeight);
-        const quadFeatureLayer = geoViewer.value.createLayer('feature', {
+        const params = {
           features: ['quad'],
           autoshareRenderer: false,
-        });
+          renderer: 'canvas',
```

Comment:
Shouldn't this say `webgl` here?

If yes, then I'd also recommend creating a boolean value (`bigImage` or something like that) and using a ternary conditional expression right in the `renderer` entry definition. This would avoid defining a default and then mutating it on (slightly) complex logic -- I think that would make the code more legible.

Reply:
Thanks, by default it didn't have a renderer and would default to webGL. I added in the default value and forgot to remove it.

I've updated to your suggestion, the formatting is caused by the annoying line length limit.